cmd/log-parser/README.md

```diff
@@ -85,5 +83,42 @@ To merge all logs:
    ```
 1. To run the program:
    ```
-   $ kata-log-parser runtime.log vc-throttler.log ksm-throttler.log
+   $ kata-log-parser kata.log
    ```
+
+### Options
+```
+   --check-only             check log files and only display output on error
+   --debug                  display debug information (requires '--output-file')
+   --error-if-file-empty    error if any files are empty
+   --error-if-no-records    error if all logfiles are empty
+   --ignore-missing-fields  do not make an error for lines with no pid, source, name, or level
+   --list-output-formats    show available formatters
+   --no-agent-unpack        do not unpack agent log entries
+   --quiet                  suppress warning messages (ignored in debug mode)
+   --strict                 do not tolerate misformed agent messages (generally caused by kernel writes to the console)
+   --output-format value    set the output format (see --list-output-formats) (default: "text")
+   --output-file value      write output to specified file
+   --help, -h               show help
+   --version, -v            print the version
+```
+
+### Advanced processing using jq
+
+[jq](https://stedolan.github.io/jq/) is a command-line JSON processor which can be combined with kata-log-parser
+to filter and fetch specific log entries
+
+#### Examples
+
+- Get only the raw guest console output:
+```
+$ kata-log-parser --ignore-missing-fields --output-format json --no-agent-unpack kata.log | jq '.Entries[] | select(.Msg=="reading guest console") | .Data.vmconsole'
+```
+- Get only the agent's unpacked log entries (and also demonstrate how to get logs from the journal directly to the parser):
```

Comment:
This example and the one below will need simpler heading names if you do convert them to headings. You can still explain the example immediately after the heading of course so we don't lose detail.